### ISSUE TITLE
Overrides (privacy, availability, color) for bidirectional sync not working as expected

### ISSUE THREAD

**Issue by A2000J** [phw198/OutlookGoogleCalendarSync#2003]
> Checklist:-
> * [x] [Upgrade](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Upgrading) to the latest release (alpha if available) ![latest release](https://img.shields.io/github/release/phw198/OutlookGoogleCalendarSync/all.svg)
> * [x] Reproduce the issue
> * [x] Complete the information below
> * [x] Provide your `OGcalsync.log` file (see [how-to](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file))

**OGCS Version**: v2.11.0.0
**Edition**: Installed
**Sync Direction**: O<->G

**Bug Description**  
With bidirectional synchronization, calendar entries that come from Outlook should get a red category in Google.
Calendar entries from Google should be given the “Private” category in Outlook and set to private.
When I work with two separate configurations it works. If I set up a two-way synchronization, new appointments are created in Google with the default category and color.

**Steps to Reproduce the Issue**  
1. Setup Configuration (s. screenshot)
2. Start Sync

**Any other information**  
Any additional information that may help troubleshoot the issue.  
:paperclip: Attach your `OGcalsync.log` file.
![BiSync_G2O](https://github.com/user-attachments/assets/0b8a281e-d268-4b78-a08c-333e08c33812)
![BiSync_O2G](https://github.com/user-attachments/assets/5a8fc8a8-ec91-452b-8784-38131cf20730)
[OGcalsync.log](https://github.com/user-attachments/files/17619578/OGcalsync.log)

**Comment by phw198** [comment]
Yes, I would expect it to work as described - if you need different colours synced in different directions (Green in Outlook, Red in Google), then you would to set up two one-way syncs, as per your screenshots. Just you need two Profiles instead of one.

**Comment by mylesmc123** [comment]
> Yes, I would expect it to work as described - if you need different colours synced in different directions (Green in Outlook, Red in Google), then you would to set up two one-way syncs, as per your screenshots. Just you need two Profiles instead of one.

Hey @phw198, 

Without a 2-way sync allowing the use of color and privacy settings, you lose the functionality of being able to delete from one calendar and have that change sync to the other. 

A way to ensure that a user can have both features simultaneously:

1. bi-directional syncing
2. calendar-based independent privacy, availability, and colour settings

Would be to set up a bi-directional profile that allows privacy and color settings for google and outlook calendars independently within the single profile and respect the independent calendar options for the "target calendar":

1. set colour/category
2. set privacy
3. set availability

![Image](https://github.com/user-attachments/assets/5e37f5aa-1417-47ec-9321-e8f27b642d09)

However, when using "target calendar" these options (particularly for me the Privacy option) are not respected. 

The real intent of this capability for me is to be able to keep my outlook events private on my google calendar and vice versa, but still keep those events in sync across both calendars. 

Is this a bug or a feature request?

**Comment by phw198** [comment]
@mylesmc123 Not sure if you're still using OGCS, but I think your use-case sits more in the "bug" category. 

:package: Please have a go with [hotfix v2.12.0.10.zip](https://github.com/user-attachments/files/25169161/v2.12.0.10.zip) and see if that works more how you need (see [instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)).
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; [CRCs](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Known-Issues#cyclic-redundancy-checks---aka-crcs): Zip = `DB333C74` Exe = `69E30774`

Let me know how it goes, if you try it!
